File: RSPAClient/Core/AtPar.POCOEntities/MT_POU_PAR_LOC_PROCESS_SCHEDULE.cs
using System;
using System.Collections.Generic;

namespace AtPar.POCOEntities
{
    public partial class MT_POU_PAR_LOC_PROCESS_SCHEDULE
    {
        public string ORG_ID { get; set; }
        public string ID { get; set; }
        public string SCHEDULE_ID { get; set; }
        public string REVIEW_CHARGES { get; set; }
        public System.DateTime LAST_UPDATE_DATE { get; set; }
        public string LAST_UPDATE_USER { get; set; }
        public string LAST_CLIENT_ADDRESS { get; set; }
        public Nullable<byte> BILLING_OPTION { get; set; }
        public short PROCESS_TYPE { get; set; }
        public Nullable<int> APP_ID { get; set; }


        //Ignore Properties
        public bool CHK_VALUE { get; set; }
        public string DESCRIPTION { get; set; }
        public int ROW_INDEX { get; set; }
        public string SOURCELOCATIONS { get; set; }
        public string DEPT_NAME { get; set; }
        public string LOW_STK_SCHEDULE_ID { get; set; }
        public string EXP_SCHEDULE_ID { get; set; }
        public string RECALL_SCHEDULE_ID { get; set; }
        public string BILLONLY_SCHEDULE_ID { get; set; }
        public short? REPLENISH_FROM { get; set; }

        //public string M_BUSINESS_UNIT { get; set; }

        //public string M_LOCATION { get; set; }

        //public string LOCATION { get; set; }

        //public string BUSINESS_UNIT { get; set; }
        //public string PERFORM_ACTION { get; set; }
        public string INV_INTERFACE_ENABLE { get; set; }
        public string BILLING_ENABLE { get; set; }



    }
}

File: RSPAClient/UserInterface/AtPar.Web/src/app/Entities/MT_POU_PAR_LOC_PROCESS_SCHEDULE.ts
export class MT_POU_PAR_LOC_PROCESS_SCHEDULE {
    public ORG_ID: string;
    public ID: string;
    public SCHEDULE_ID: string;
    public REVIEW_CHARGES: string;
    public LAST_UPDATE_DATE:  Date;
    public LAST_UPDATE_USER: string;
    public LAST_CLIENT_ADDRESS: string;
    public BILLING_OPTION?: number;
    public PROCESS_TYPE: number;
    public APP_ID?: number;
    public REPLENISH_FROM: any;
    public OLDREPLENISH_FROM: any;
    //public REPLENISH_FROM?: any;

    public CHK_VALUE: boolean = false;
    public DESCRIPTION: string;   
    public ROW_INDEX?: number;
    public SOURCELOCATIONS: string;
    public OLDSOURCELOCATIONS: string;
    public DEPT_NAME: string;

    public LOW_STK_SCHEDULE_ID: string;
    public EXP_SCHEDULE_ID: string;
    public RECALL_SCHEDULE_ID: string;
    public BILLONLY_SCHEDULE_ID: string;
    public INV_INTERFACE_ENABLE: string;
    public BILLING_ENABLE: string;
    public BlnReviewChargeValue: boolean = false;
    public blnBillScheduleId: boolean = true;
    public chkBillOption: boolean = true;
    public blnSplice: boolean = true;
}